pkgs/by-name/pe/peergos/package.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchurl
+, jre
+, makeWrapper
+}:
+
+let
+  version = "0.14.1";
+  peergos = fetchurl {
+    url = "https://github.com/Peergos/web-ui/releases/download/v${version}/Peergos.jar";
+    hash = "sha256-oCsUuFxTAL0vAabGggGhZHaF40A5TLfkT15HYPiKHlU=";
+  };
+in stdenv.mkDerivation rec {
+  inherit version;
+  pname = "peergos";
+  meta = with lib; {
+    description = "A p2p, secure file storage, social network and application protocol";
+    homepage = "https://peergos.org/";
+    # peergos have agpt3 license, peergos-web-ui have gpl3, both are used
+    license = [licenses.agpl3Only licenses.gpl3Only];
+    platforms = platforms.all;
+    maintainers = with maintainers; [ raspher ];
+  };
+
+  dontUnpack = 1;
```

```suggestion
  dontUnpack = true;
```
I guess we also want `dontBuild`.